addons/hr/views/hr_job_views.xml

```diff
@@ -19,15 +19,14 @@
                             <page string="Recruitment" name="recruitment_page">
                                 <group>
                                     <group name="recruitment">
-                                        <field name="company_id" options="{'no_create': True}" groups="base.group_multi_company"/>
```

Comment:
As we discussed, we can't just remove the field from the view in stable, cause that might break customization. Instead what we could do is add `invisible="1"` to it :bulb: 

Applies to this and all the other places that we remove field from the view